src/user_group.rs

```diff
@@ -98,6 +98,16 @@ impl UserGroup {
         }
     }
 
+    // Return group identities, convert exported groups into identities
+    // A group can also be part of similar package or a different package
+    pub fn identities(&self) -> Vec<(String, String)> {
```

Comment:
Here we are modelling an identity as `(String, String)`. In line 196 we are modelling an identity as `String`. Lets create a struct/type called `fpm::UserIdentity` and use it everywhere.